owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/shares/ShareXMLParser.java

```diff
@@ -36,413 +37,407 @@
+
+	private static final String NODE_DATA = "data";
+	private static final String NODE_ELEMENT = "element";
+	private static final String NODE_ID = "id";
+	private static final String NODE_ITEM_TYPE = "item_type";
+	private static final String NODE_ITEM_SOURCE = "item_source";
+	private static final String NODE_PARENT = "parent";
+	private static final String NODE_SHARE_TYPE = "share_type";
+	private static final String NODE_SHARE_WITH = "share_with";
+	private static final String NODE_FILE_SOURCE = "file_source";
+	private static final String NODE_PATH = "path";
+	private static final String NODE_PERMISSIONS = "permissions";
+	private static final String NODE_STIME = "stime";
+	private static final String NODE_EXPIRATION = "expiration";
+	private static final String NODE_TOKEN = "token";
+	private static final String NODE_STORAGE = "storage";
+	private static final String NODE_MAIL_SEND = "mail_send";
+	private static final String NODE_SHARE_WITH_DISPLAY_NAME = "share_with_displayname";
+	private static final String NODE_SHARE_WITH_ADDITIONAL_INFO = "share_with_additional_info";
+	private static final String NODE_NAME = "name";
+
+	private static final String NODE_URL = "url";
+
+	private static final String TYPE_FOLDER = "folder";
+
+	private static final int SUCCESS = 200;
+	private static final int ERROR_WRONG_PARAMETER = 400;
+	private static final int ERROR_FORBIDDEN = 403;
+	private static final int ERROR_NOT_FOUND = 404;
+
+	private String mStatus;
+	private int mStatusCode;
+	private String mMessage;
+
+	// Getters and Setters
+	public String getStatus() {
+		return mStatus;
+	}
+
+	public void setStatus(String status) {
+		this.mStatus = status;
+	}
+
+	public int getStatusCode() {
+		return mStatusCode;
+	}
+
+	public void setStatusCode(int statusCode) {
+		this.mStatusCode = statusCode;
+	}
+
+	public String getMessage() {
+		return mMessage;
+	}
+
+	public void setMessage(String message) {
+		this.mMessage = message;
+	}
+
+	// Constructor
+	public ShareXMLParser() {
+		mStatusCode = -1;
+	}
+
+	public boolean isSuccess() {
+		return mStatusCode == SUCCESS;
+	}
+
+	public boolean isForbidden() {
+		return mStatusCode == ERROR_FORBIDDEN;
+	}
+
+	public boolean isNotFound() {
+		return mStatusCode == ERROR_NOT_FOUND;
+	}
+
+	public boolean isWrongParameter() {
+		return mStatusCode == ERROR_WRONG_PARAMETER;
+	}
+
+	/**
+	 * Parse is as response of Share API
+	 * @param is
+	 * @return List of ShareRemoteFiles
+	 * @throws XmlPullParserException
+	 * @throws IOException
+	 */
+	public ArrayList<OCShare> parseXMLResponse(InputStream is) throws XmlPullParserException,
+			IOException {
+
+		try {
+			// XMLPullParser
+			XmlPullParserFactory factory = XmlPullParserFactory.newInstance();
+			factory.setNamespaceAware(true);
+
+			XmlPullParser parser = Xml.newPullParser();
+			parser.setFeature(XmlPullParser.FEATURE_PROCESS_NAMESPACES, false);
+			parser.setInput(is, null);
+			parser.nextTag();
+			return readOCS(parser);
+
```

Comment:
What is the criteria of this long indentation?

Reply:
Same here